archibald/cogs/links.json

```diff
@@ -3,6 +3,6 @@
         "-cshub": "https://lausta.notion.site/CS-Hub-6e7cae889f844cb59ae5f1809c88e553",
         "-links": "https://csdiscdocs.gitbook.io/vu-cs-amst-guide/",
         "-maths": "https://lausta.notion.site/Maths-courses-suggested-for-an-ambitious-CS-student-d3da2ccb468546629da227cab9188d58?pvs=4",
-        "-finance": "https://docs.google.com/document/d/19bZOmYB75NXlQ6l8PE40R_UcyZ-VSAyYoJPfofmWaYo/edit",
+        "-finance": "https://docs.google.com/document/d/19bZOmYB75NXlQ6l8PE40R_UcyZ-VSAyYoJPfofmWaYo/edit"
```

Comment:
these should be `cshub` instead of `-cshub` since the `-` prefix is handled earlier